packages/features/insights/hooks/useInsightsColumns.tsx

```diff
@@ -68,6 +68,32 @@ export const useInsightsColumns = ({
           return dataTableFilter(cellValue, filterValue);
         },
       }),
+      columnHelper.accessor("bookingUid", {
+        id: "bookingUid",
+        header: t("routing_form_insights_booking_uid"),
+        size: 100,
+        enableColumnFilter: false,
+        enableSorting: false,
+        cell: (info) => {
+          const bookingUid = info.getValue();
+          if (!bookingUid) return null;
+          return <CopyButton label={bookingUid} value={bookingUid} />;
+        },
+      }),
+      columnHelper.accessor("bookingUid", {
+        id: "bookingLink",
+        header: t("routing_form_insights_booking_link"),
+        size: 100,
+        enableColumnFilter: false,
+        enableSorting: false,
+        cell: (info) => {
+          const bookingUid = info.getValue();
+          if (!bookingUid) return null;
+          const displayedUrl = `${location.host}/booking/${bookingUid}`;
+          const bookingUrl = `https://${displayedUrl}`;
+          return <CopyButton label={displayedUrl} value={bookingUrl} />;
```

Comment:
Can someone validate if this is a correct logic to build booking url, or not? Based on other similar code, it's probably right, but I'd like to make sure about it, especially the `location.host` part.

Reply:
Thanks Keith!